### DISCUSSION TITLE
User authentication with URLs

### DISCUSSION THREAD

**Discussion by jashan8** [Budibase/budibase#8455] [Help] [answered]
Hi there,

Is it possible to authenticate the user to the Budibase Application with a URL that contains their access key or user id?

The requirement for the application is that users will log in with unique URLs, and with that, they will bypass the username/password at the login screen.

Thank you

**Comment by melohagan** [comment, marked as answer]
Hey @jashan8 

This is not possible. We do support OAuth if that helps at all: https://docs.budibase.com/docs/authentication-and-sso
